src/main/java/mate/academy/service/AuthenticationService.java

```diff
@@ -11,6 +13,19 @@
      * Return false in any other cases.
      */
     public boolean login(String email, String password) {
-        return false;
+        if (email == null || password == null) {
+            return false;
+        } else {
+            UserService iuserService = new UserService();
+            User iuser = iuserService.findByEmail(email);
+            if (iuser == null) {
+                return false;
+            } else {
+                if (email.equals(iuser.getEmail()) && password.equals(iuser.getPassword())) {
+                    return true;
+                }
+            }
+            return false;
+        }
```

Comment:
Please, don't complicate if-else constructions
```suggestion
        UserService userService = new UserService();
        User user = userService.findByEmail(email);
        if (user != null) {
            return user.getPassword().equals(password);
        } 
        return false;
```